docs/modules/commons/pages/expressions.adoc

```diff
@@ -419,3 +419,29 @@ When I initialize Scenario variable `listOfMaps` with values:
 |3  |
 Then `1-2-3` is = `#{evalGroovy(return listOfMaps.collect{it['key']}.sort().join('-'))}`
 ----
+
+== Null generation
+
+=== `null`
+
+Generates the `null`. Unlike the expression `eval(null)` generates not the string with `null` value, but the https://docs.oracle.com/javase/specs/jls/se11/html/jls-3.html#jls-3.10.7[special literal] of the null type.
+
+[source, subs="+quotes"]
+----
+#{null}
+----
+
+NOTE: Null generation expression doesn't work in examples tables.
```

Comment:
```suggestion
```
I guess with the proposal below it would possible to use the expression in ExamplesTable-s